content/en/flux/use-cases/helm.md

```diff
@@ -47,25 +54,26 @@ helm install my-traefik traefik/traefik \
 
 Flux client:
 
+Clone your bootstrap repository to your local machine and create a `traefik` directory in your bootstrap path.
+
 ```sh
-flux create source helm traefik --url https://helm.traefik.io/traefik --namespace traefik --export > /flux/boot/traefik/helmrepo.yaml
+flux create source helm traefik --url https://helm.traefik.io/traefik --namespace traefik --export > traefik/helmrepo.yaml
 flux create helmrelease my-traefik --chart traefik \
   --source HelmRepository/traefik \
   --chart-version 9.18.2 \
   --namespace traefik \
-  --export > /flux/boot/traefik/helmrepo.yaml
+  --export > traefik/helmrepo.yaml
```

Comment:
Can you please test this with Kind, I guess the HR will fail due to the lack of LB support.

Reply:
I propose we replace traefik with nginx, and then we can use the HR from here which is e2e tested on Kind. https://github.com/fluxcd/flux2-kustomize-helm-example/blob/main/infrastructure/controllers/ingress-nginx.yaml

Reply:
@stefanprodan you are right
```
 Helm install failed: timed out waiting for the condition                                                                                                                                │
                                                                                                                                                                                              │
      Last Helm logs:                                                                                                                                                                         │
                                                                                                                                                                                              │
      creating 1 resource(s)                                                                                                                                                                  │
      CRD traefikservices.traefik.containo.us is already present. Skipping.                                                                                                                   │
      creating 5 resource(s)                                                                                                                                                                  │
      beginning wait for 5 resources with timeout of 5m0s                                                                                                                                     │
      Service does not have load balancer ingress IP address: flux-system/my-traefik
```

I will update it to use nginx

Reply:
Nginx also has the same problem. Should we use podinfo?

Reply:
Nginx does not have this issue, see the values used in the link I posted above

Reply:
In the usecase guide, values are introduced in a later section after the installation. 
So I was thinking we could start with something that just works without tweaking the values first

Reply:
OK lets' use Flagger.